homeworks/rostyslav-reshmed/homework-4/script-2.js

```diff
@@ -0,0 +1,28 @@
+const fruits = ['banana', 'apple', 'orange', 'watermelon', 'apple', 'orange', 'grape', 'apple'];
+//a. повернути массив унікальних значень
+function getUniqueFruit (value, index, self) {
+    return self.indexOf(value) === index;
+}
+const unigue = fruits.filter(getUniqueFruit);
+console.log(unigue);
+
+//b. очистити масив
+function getEmptyArr (array) {
+    fruits.splice(0,fruits.length);
+}
+getEmptyArr(fruits);
+console.log(fruits);
+
+//с. повернути рандомне значення масиву
+const fruit = ['banana', 'apple', 'orange', 'watermelon', 'apple', 'orange', 'grape', 'apple'];
+
+function getRandomFruit (array) {
+    return fruit[(Math.floor(Math.random() * (fruit.length)))];
+}
+console.log(getRandomFruit(fruit))
+
+//d. Перетворити массив на об'єкт.
+function getObj (array) {
+    return {...fruit};
```

Comment:
Тут ти перетворюєш fruit, хоча в тебе аргумент array
```suggestion
    return {...array};
```